Fix International Calling Connoisseur hints (#2962)
exercism/java: exercises/concept/international-calling-connoisseur/.docs/hints.md

## 5. Find a country's dialing code

> - There is a [way][map-entry-set-docs] to get an iterable collection of entries in a map, which allows you to search through both keys and values.

Reviewer: Thanks for taking this on! My only thought with the wording is that it sounds like we could use `entrySet()` to go through the keys independently of the values (which you could, but then there are other methods for that). What do you think of describing it as going through the key-value pairs (the introduction describes maps in terms of this pair)? I think it could be something like this 👇🏻  

```suggestion
- There is a [way][map-entry-set-docs] to get an iterable collection of entries in a map, which allows you to go through the key-value pairs.
```
